files/scripts/selinux/trivalent/trivalent.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
I don't see a reason for this to be `#!/usr/bin/env bash` if we aren't using any bashisms - while all our images ship with bash, might as well just use posix sh for this